app/src/main/report/actionCreators/reportPageActionCreators.ts

```diff
@@ -14,9 +14,14 @@ export const reportPageActionCreators = {
                 type: ReportTypeKeys.REPORT_VERSION_DETAILS_FETCHED,
                 data: null
             } as ReportVersionDetailssFetched);
+            dispatch({
+                type: ReportTypeKeys.REPORT_VERSION_CHANGELOG_FETCHED,
+                data: null
+            } as ReportVersionChangelogFetched);
 
             dispatch(reportActionCreators.getReportVersions(props.report));
             dispatch(reportActionCreators.getVersionDetails(props.report, props.version));
+            dispatch(reportActionCreators.getVersionChangelog(props.report, props.version));
```

Comment:
At the moment this is going to throw an error if the user is not a report reviewer so this would cause an error message for all normal users visiting a report page. It should probably be dispatched by the `ReportChangelog` component itself, seeing as only reviewers will be given access to that.